RULES.md

```diff
@@ -1,373 +1,248 @@
-| [E053](#E053) | Trip frequencies overlap |
-| [E054](#E054) | Block trips must not have overlapping stop times |
-| [E055](#E055) | Mismatching feed and agency language fields |
-| [E056](#E056) | Missing `calendar_dates.txt` and `calendar.txt` files |
-| [E057](#E057) | Decreasing `shape_dist_traveled` in `stop_times.txt` |
-| [E058](#E058) | Decreasing `shape_dist_traveled` in `shapes.txt` |
-| [E059](#E059) | GTFS dataset too big |
-| [E060](#E060) | Fatal internal error -- please report |
-| [E061](#E061) | Out of memory |
-
-### Table of Warnings
-
-| Warning ID    | Warning Title             |
-|---------------|---------------------------|
-| [W001](#W001) | Input zip archive contains folder | 
-| [W002](#W002) | Non standard field name | 
-| [W003](#W003) | Non ascii or non printable char in `id` | 
-| [W004](#W004) | Extra `file` found | 
-| [W005](#W005) | Route short name too long |
-| [W006](#W006) | Missing route short name |
-| [W007](#W007) | Missing route long name |
-| [W008](#W008) | Route long name contains short name | 
-| [W009](#W009) | Dataset should cover at least the next 30 days of service | 
-| [W010](#W010) | `feed_end_date` should be provided if `feed_start_date` is provided | 
-| [W011](#W011) | `feed_start_date` should be provided if `feed_end_date` is provided | 
-| [W012](#W012) | Optional csv file is empty | 
-| [W014](#W014) | Duplicate `routes.route_long_name` | 
-| [W015](#W015) | Duplicate `routes.route_short_name` | 
-| [W016](#W016) | Duplicate combination of fields `route_long_name` and `routes.route_short_name` | 
+Rules are declared in the `Notice` modules: 
+- [Notice related to CSV structure issues](https://github.com/MobilityData/gtfs-validator/tree/master/core/src/java/org/mobilitydata/gtfsvalidator/notice) 
+- [Notices related to GTFS semantic rules issues](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/notice). 
+ 
+Below are details of currently implemented rules.
+** Please note that the error ID naming conventions changed in `v2` to make contributions of new rules easier by reducing the likelihood of conflicting IDs during parallel development. The table below indicates the ID used in gtfs-validator `v2` and higher in the first column and the legacy ID in the second column.** 
+
+### Table of Notices [WIP]
+
+| Error ID (v2.0+) | Error ID (v1.x) 	| Error Title                                                             	|
+|--------------------------	|--------------------	|-------------------------------------------------------------------------	|
+| [`MissingRequiredFieldError`](#MissingRequiredFieldError) | [E015](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/RULES.md#E015)      	| Missing required `field`                                                	|
+| [`DuplicatedColumnNotice`](#DuplicatedColumnNotice)  |     ❌   	            | Duplicated column                                                      	|
```

Comment:
I would also get rid of the red X and just leave the column blank when the rule didn't exist in v1.x.